tests/test_apiclient.py

```diff
@@ -389,6 +389,16 @@ def test_get_search_url(self, search_client):
     def test_get_aggregations_url(self, search_client):
         assert search_client.get_aggregations_url('g-cloud-9') == 'http://baseurl/g-cloud-9/services/aggregations'
 
+    @pytest.mark.parametrize('search_api_url, expected_index',
+                             (
+                                 ('http://localhost/g-cloud-8/services/search', 'g-cloud-8'),
+                                 ('http://localhost/g-cloud-9/services/search', 'g-cloud-9'),
+                                 ('https://search-api.preview.marketplace.team/g-cloud-9/services/search', 'g-cloud-9'),
+                                 ('https://search-api.preview.marketplace.team/g-cloud-8/services/search', 'g-cloud-8')
+                             ))
```

Comment:
Is it worth adding a case for a non-matching URL path that returns `None`?

Reply:
Yes, it is. Thanks.